electron/ipc/docker.ts

```diff
@@ -11,12 +11,7 @@ export interface PullImageProgress {
   id: string;
 }
 
-let handlersBound = false;
-
-export function bindDockerIpc(win: BrowserWindow) {
-  if (handlersBound) return; // Prevent duplicate registrations
-  handlersBound = true;
-
+export function bindDockerIpc({ win }: { win?: BrowserWindow }) {
```

Comment:
Destructure this way will lose the reference. Imagine:

```javascript
ref = {
  count: 1
}

function bar({count}) {
  setInterval(() => console.log('Count is', count), 1000);
}

// This will print count 1 every second

ref.count = 2;
// Now I update new value, but as you can see, it will still print 1 every second instead of print 2
```